packages/site/src/assets/JobberLogo.svg.tsx

```diff
@@ -1,6 +1,6 @@
 export const JobberLogo = () => {
   return (
-    <svg>
+    <svg width="auto" height="24">
```

Comment:
Address the large implicit box this SVG was creating in the layout
![image](https://github.com/user-attachments/assets/50d43f9e-6d5d-4152-b23f-ece2d28117c9)
 
(Before:)
![image](https://github.com/user-attachments/assets/8abc2738-b6a2-42bf-976c-fe1e2e9b06d1)